test/runtests.jl

```diff
@@ -176,7 +176,7 @@ end
 
 @testset "Gamma IdentityLink" begin
     gm10 = fit(GeneralizedLinearModel, lot1 ~ 1 + u, clotting, Gamma(), IdentityLink(),
-        convTol=1e-8)
```

Comment:
Hmm, I'm not sure which comment you refer to exactly, but in general PRs should avoid making changes to unrelated parts of the code, and this one in particular does not use a correct alignment. Please revert whitespace changes outside of the code you need to change.